README.md

```diff
@@ -28,6 +28,12 @@ Currently tests are only run for the `web3` api intialisation found inside `src/
 npm run test
 ```
 
+## Running CRA example
+
+Build the library with `npm run build`
+
+Inside `./examples/cra-test`, install dependencies `npm install` and run the example `npm start`
```

Comment:
There's not such dir in this PR `./examples/cra-test` I think that is in another branch...

Reply:
yep